static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -6,11 +6,24 @@ import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import once from 'lodash/once';
 import { prebid } from 'commercial/modules/prebid/prebid';
 
+const isGoogleWebPreview: () => boolean = () => {
+    try {
+        return navigator.userAgent.indexOf('Google Web Preview') > -1;
+    } catch (exception) {
+        return false;
+    }
+};
+
+if (!isGoogleWebPreview()) {
+    require('prebid.js/build/dist/prebid'); // eslint-disable-line global-require
```

Comment:
For posterity, why use [curl](https://github.com/cujojs/curl)'s `require` instead of [dynamic `import(...)`](https://webpack.js.org/guides/code-splitting/#dynamic-imports) here?

Reply:
@SiAdcock You can't use a dynamic import here because Jest doesn't understand it and all the tests fail.

Reply:
I see! Would it be possible to install the [`dynamic-import-node`](https://github.com/airbnb/babel-plugin-dynamic-import-node) plugin for Babel and include it in the [test plugins](https://github.com/guardian/frontend/blob/master/babel.config.js#L42-L43) array? Only asking because `require` is not idiomatic, and is only supported in the client for legacy integrations such as interactives.

Reply:
Yes, I can try that tomorrow! Thanks Simon